### PR TITLE
Add application type in Headers

### PR DESCRIPTION
**Pull request by vitPinchuk**
Resolves: [#60 ](https://github.com/VolkovLabs/volkovlabs-rss-datasource/issues/60)

### CODE REVIEW

File: src/api/api.ts [hunk 1]
```diff
@@ -52,6 +52,9 @@ export class Api {
     const response = await lastValueFrom(
       getBackendSrv().fetch({
         method: 'GET',
+        headers: {
+          Accept: 'application/rss+xml;charset=UTF-8',
```
Comment (asimonok) [inline src/api/api.ts:56]: Please remove charset from the header. Just `application/rss+xml`

**Review by asimonok — CHANGES_REQUESTED** [review]

**Review by asimonok — APPROVED** [review]
Looks good!

@mikhail-vl ready for review

File: src/api/api.ts [hunk 2]
```diff
@@ -52,6 +52,9 @@ export class Api {
     const response = await lastValueFrom(
       getBackendSrv().fetch({
         method: 'GET',
+        headers: {
+          Accept: 'application/rss+xml',
```
Comment (mikhail-vl) [inline src/api/api.ts:56]: Data Source supports RSS and Atom feedback. For Atom the Content-Type is `application/atom+xml`. With this change we may break it?

**Review by mikhail-vl — APPROVED** [review]
Looks good!

**Pull request merged**